BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-1.0.25-prerelease-00420-01
\ No newline at end of file
+1.0.25-prerelease-00424-03
```

Comment:
version 00425-03 causes build errors, I am going to merge 00424-03 which works and then investigate the errors for 425-03.